django/contrib/admin/templates/admin/base.html

```diff
@@ -24,7 +24,7 @@
 
 <body class="{% if is_popup %}popup {% endif %}{% block bodyclass %}{% endblock %}"
   data-admin-utc-offset="{% now "Z" %}">
-
+  <a href="#content" class="skip-to-content-link">Skip to main content</a>
```

Comment:
This will need to be translate-able,

```suggestion
  <a href="#content" class="skip-to-content-link">{% translate 'Skip to main content' %}</a>
```

Reply:
Thank you!

Change applied!